iron-overlay-behavior.html

```diff
@@ -341,9 +341,7 @@
       }
 
       this.fire('iron-overlay-opened');
-
-      this._squelchNextResize = true;
-      this.async(this.notifyResize);
+      this.async(this.refit);
```

Comment:
`notifyResize` has additional side-effects. Children of the overlay have their resize handlers called as well. So, even if `refit` should only be called once, `notifyResize` still needs to be called to maintain implementation parity.